zavdannya3.rb

```diff
@@ -0,0 +1,35 @@
+array = %w['o', 'o', 'k']
```

Comment:
you dont need `"" `and ` , ` if you use %w 
That mean what `%w[o o k]`  will have output  `['o', 'o', 'k']`

Reply:
but that code will give you `["'o',", "'o',", "'k'"]`

Reply:
Thanks for advice!